spec/bolt/applicator_spec.rb

```diff
@@ -73,8 +73,9 @@
       end
 
       targets = [Bolt::Target.new('node1'), Bolt::Target.new('node2'), Bolt::Target.new('node3')]
-      results = targets.map { |target| Bolt::Result.new(target) }
-      allow_any_instance_of(Bolt::Transport::SSH).to receive(:batch_task).and_return(*results)
+      allow_any_instance_of(Bolt::Transport::SSH).to receive(:batch_task) do |_, batch|
```

Comment:
This test was passing before because the error when we returned the result for the wrong target was ignored. Now without `_catch_errors=true`, it gets thrown.